themes/src/main/resources-community/theme/base/login/messages/messages_ka.properties

```diff
@@ -0,0 +1,477 @@
+identityProviderAlreadyLinkedMessage=ფედერაციული იდენტიფიკატორი, რომელიც {0}-მა დააბრუნა, უკვე მიბმულია სხვა მომხმარებელზე.
+confirmAccountLinking=დაადასტურეთ ანგარიშის {0} იდენტიფიკატორის მომწოდებლიდან {1} მიბმა თქვენს ანგარიშზე.
+confirmEmailAddressVerification=დაადასტურეთ ელფოსტის მისამართის {0} სისწორე.
+requiredAction.CONFIGURE_RECOVERY_AUTHN_CODES=აღდგენის კოდების გენერაცია
+confirmExecutionOfActions=შეასრულეთ შემდეგი ქმედებები
+backToApplication=&laquo; აპლიკაციაზე დაბრუნება
+brokerLinkingSessionExpired=მოთხოვნილია ბროკერის ანგარიშის მიბმა, მაგრამ მიმდინარე სესია უკვე არასწორია.
+differentUserAuthenticated=ავთენტიკაცია უკვე გაიარეთ სხვა მომხმარებლით "{0}" ამ სესიაში. ჯერ უნდა გახვიდეთ.
+requiredAction.TERMS_AND_CONDITIONS=წესები და პირობები
+acrNotFulfilled=ავთენტიკაციის მოთხოვნები დაკმაყოფილებული არაა
+requiredAction.CONFIGURE_TOTP=OTP-ის მორგება
+requiredAction.webauthn-register-passwordless=Webauthn-ზე რეგისტრაცია უპაროლოდ
+clientCertificate=X509 კლიენტის სერტიფიკატი:
+invalidTokenRequiredActions=ამ ბმულში მოთხოვნილი ქმედებები არასწორია
+internalServerError=აღმოჩენილია სერვერის შიდა შეცდომა
+noCertificate=[სერტიფიკატის გარეშე]
+
+
+pageNotFound=გვერდი ვერ ვიპოვე
+openshift.scope.user_full=სრული წვდომა
+console-verify-email=საჭიროა, გადაამოწმოთ თქვენი ელფოსტის მისამართი.   გამოგიგზავნეთ ელფოსტა {0}-ზე, რომელიც გადამოწმების კოდს შეიცავს.   შეიყვანეთ ეს კოდი ქვემოთ ველში.
+saml.artifactResolutionServiceInvalidResponse=არტეფაქტის გადაწყვეტა შეუძლებელია.
+openshift.scope.user_check-access=მომხმარებლის წვდომის ინფორმაცია
+openshift.scope.list-projects=პროექტების სია
+otp-help-text=შეიყვანეთ გადამოწმების კოდი ავთენტიკატორი აპლიკაციიდან.
+saml.post-form.js-disabled=JavaScript გათიშულია. გირჩევთ, ჩართოთ ის. გასაგრძელებლად დააწექით ღილაკს ქვემოთ. 
+password-display-name=პაროლი
+
+#authenticators
+otp-display-name=ავთენტიკატორი აპლიკაცია
+otp-reset-description=რომელი OTP-ის კონფიგურაციის წაშლა გნებავთ?
+auth-recovery-authn-code-form-help-text=შეიყვანეთ აღდგენის ავთენტიკაციის სიიდან, რომელიც ადრე გადმოგეცათ.
+recovery-code-config-warning-title=ეს ავთენტიკაციის კოდები აღარასდროს გამოჩნდება, როცა ამ გვერდიდან გახვალთ
+
+# Recovery Codes
+auth-recovery-authn-code-form-display-name=აღდგენის ავთენტიკაციის კოდი
+auth-recovery-code-info-message=შეიყვანეთ მითითებული აღდგენის კოდი.
+recovery-codes-error-invalid=არასწორი აღდგენის ავთენტიკაციის კოდი
+auth-recovery-code-prompt=აღდგენის კოდი #{0}
+auth-recovery-code-header=შესვლა აღდგენის ავთენტიკაციის კოდით
+recovery-code-config-header=აღდგენის ავთენტიკაიციის კოდები
+recovery-codes-print=დაბეჭდვა
+recovery-codes-copy=კოპირება
+recovery-codes-download-file-description=აღდგენის კოდები ერთჯერადი საკვანძო კოდებია, რომელიც საშუალებას გაძლევთ, შეხვიდეთ თქვენს ანგარიშში, თუ ავთენტიკატორთან წვდომა დაკარგეთ.
+recovery-codes-label-default=აღდგენის კოდები
+recovery-codes-download=გადმოწერა
+
+# WebAuthn
+webauthn-display-name=საკვანძო გასაღები
```

Comment:
"Passkey" is an industry term, and I'd prefer not to translate it. I ass this to this message, and it applies to all messages that contain "Passkey".

See https://fidoalliance.org/passkeys/ for more information about Passkey. "Passkeys" (the plural) usually refers to the method of using the standard and the procedure. We use the singular "Passkey" to refer to a single key pair of a user.

Reply:
OK, I replaced every Passkey with latin variant :)